backstage-plugin/plugins/open-dora/src/locales/en/translation.json

```diff
@@ -0,0 +1,25 @@
+{
+  "deployment_frequency": {
+    "labels": {
+      "deployment_frequency": "Deployment Frequency",
+      "deployment_frequency_average": "Deployment Frequency Average"
+    },
+    "overall_labels": {
+      "lt-6month": "Less then 6 months",
```

Comment:
I think it's nicer to use more verbose descriptions like the ones that DevLake uses: https://devlake.apache.org/docs/Metrics/DeploymentFrequency#how-is-it-calculated
On-demand, 
Between once per week and once per month,
Between once per month and once every 6 months,
Fewer than once per six months